.cloudbuild/library_generation/library_generation.Dockerfile

```diff
@@ -47,7 +61,16 @@ RUN source /src/utils/utilities.sh \
 ENV DOCKER_GRPC_LOCATION="/grpc/protoc-gen-grpc-java-${GRPC_VERSION}-${OS_ARCHITECTURE}.exe"
 ENV DOCKER_GRPC_VERSION="${GRPC_VERSION}"
 
-# use python 3.11 (the base image has several python versions; here we define the default one)
+
+# we transfer gapic-generator-java from the previous stage.
+# here we redeclare this env var since they are not preserved between stages
+ENV DOCKER_GAPIC_GENERATOR_VERSION="2.43.1-SNAPSHOT" 
```

Comment:
Can we remove this version? Since we know the location from the previous stage, can we just use whatever jar is there?

Reply:
https://github.com/googleapis/sdk-platform-java/blob/a5c18b2920bfd349119ed95a75da785d1fa3f23a/library_generation/utils/utilities.sh#L182-L187

Reply:
I see. For protoc, we tried to make it more flexible since we do have the need to test different protoc versions frequently. I don't think we have this need for gapic-generator-java. Hence I think it's better to make the implementation simple. It could be a separate PR also.

Reply:
ok, I removed gapic_generator_version from the scripts. We will now always assume `DOCKER_GAPIC_GENERATOR_LOCATION` has the jar we need. We don't care about the generator version anymore.